db/migrate/20190901150206_add_accepted_to_events_attendees.rb

```diff
@@ -0,0 +1,6 @@
+class AddAcceptedToEventsAttendees < ActiveRecord::Migration[5.2]
+  def change
+    add_column :events_attendees, :accepted, :boolean, default: false
+    add_column :events_attendees, :declined, :boolean, default: false
```

Comment:
You don't need to change this but with just one field is enough. Just one boolean value (`accepted`, for example) can handle both states.